cmd/kubelet/app/server.go

```diff
@@ -1061,7 +1063,9 @@ func CreateAndInitKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	nodeLabels map[string]string,
 	seccompProfileRoot string,
 	bootstrapCheckpointPath string,
-	nodeStatusMaxImages int32) (k kubelet.Bootstrap, err error) {
+	nodeStatusMaxImages int32,
+	hostname string,
```

Comment:
adding hostname when we already have a hostnameOverride parameter is a little crazy-making

Reply:
Yeah , but only paasing nodename is not clean. 
i will close this pr, thanks ! @liggitt